tests/unit/core/scan/test_scan_context.py

```diff
@@ -1,12 +1,21 @@
+import os
 from pathlib import Path
 from typing import Union
+from unittest import mock
 
 from ggshield.core.scan.scan_context import ScanContext
 from ggshield.core.scan.scan_mode import ScanMode
 from tests.repository import Repository
 
 
 class TestScanContextRepositoryURL:
+    DOMAIN = "https://github.com"
+    SLUG = "expected/repository"
+    REPO_URL = f"{DOMAIN}/{SLUG}"
+    FULL_REPO_URL = "https://user:password@github.com:84/expected/repository.git"
+    FAKE_URL = "https://github.com/other/repository"
+    EXPECTED_HEADER = "github.com/expected/repository"
+
```

Comment:
nit: Those could be moved outside the class to avoid the need for the `self.` prefix.

Actually the class itself is not necessary: pytest is happy to use standalone functions. Not having a class saves indentation levels.